### PR TITLE
bg-0002: Add basic page structure and routing

### PR DESCRIPTION
**Pull request by chemmy**

### CODE REVIEW

File: src/scenes/app.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+function App() {
+  return (
+    <div className="App">
+      <Layout>
+        <BrowserRouter>
+          <Header />
+          <Content>
+            <Switch>
+              <Route exact path="/" component={Home} />
+              <Route exact path="/library" component={Library} />
```
Comment (ManrickCapotolan) [inline src/scenes/app.tsx:22]: create constants for path?

File: src/components/layout/header/header.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    setActiveMenu(path[1]);
+  }, [location]);
+
+  const onMenuItemClick = (e: MenuInfo) => {
+    setActiveMenu(e.key as string);
+  };
+
+  return (
+    <AntdHeader style={{ position: 'fixed', zIndex: 1, width: '100%' }}>
+      <div className="logo">
```
Comment (ManrickCapotolan) [inline src/components/layout/header/header.tsx]: create logo component? as this will most likely be used again

File: src/scenes/app.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Layout } from 'antd';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+
+import Header from '../components/layout/header/header';
```
Comment (ManrickCapotolan) [inline src/scenes/app.tsx]: try setting up absolute import path

File: src/components/layout/header/header.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  const [activeMenu, setActiveMenu] = useState('home');
+  const location = useLocation();
+
+  useEffect(() => {
+    const path = location.pathname.split('/');
+    setActiveMenu(path[1]);
+  }, [location]);
+
+  const onMenuItemClick = (e: MenuInfo) => {
+    setActiveMenu(e.key as string);
```
Comment (ManrickCapotolan) [inline src/components/layout/header/header.tsx:25]: use `e.item.key` so we don't need to cast it

Comment (chemmy) [reply src/components/layout/header/header.tsx:25]: Error: Property 'key' does not exist on type 'ReactInstace'

**Review by ManrickCapotolan — APPROVED** [review]

**Pull request merged**